constraint/pkg/client/client_test.go

```diff
@@ -330,6 +347,18 @@ func TestAddTemplate(t *testing.T) {
 			Template:      badRegoTempl,
 			ErrorExpected: true,
 		},
+		{
+			Name:          "Bad Arity",
+			Handler:       &badHandler{Name: "h1", HasLib: true},
+			Template:      badArityTempl,
+			ErrorExpected: true,
+		},
+		{
+			Name:          "Missing Rule",
+			Handler:       &badHandler{Name: "h1", HasLib: true},
+			Template:      missingRuleTempl,
+			ErrorExpected: true,
+		},
```

Comment:
Should we also have a positive test for case`violation[r]`?

Reply:
Taken care of in `Good Template`